src/Orc.NuGetExplorer/Services/PackageInstallationService.cs

```diff
@@ -271,12 +272,20 @@ public async Task<InstallerResult> InstallAsync(
                     throw Log.ErrorAndCreateException<IncompatiblePackageException>($"Package {package} incompatible with project target platform {targetFramework}");
                 }
 
-                // Step 5. Build install list using NuGet Resolver and select available resources. 
-                // Track packages which already installed and make sure only one version of package exists
-                var resolver = new Resolver.PackageResolver();
-                var availablePackagesToInstall = await resolver.ResolveWithVersionOverrideAsync(resolverContext, project, DependencyBehavior.Highest,
-                    (project, conflict) => _fileSystemService.CreateDeleteme(conflict.PackageIdentity.Id, project.GetInstallPath(conflict.PackageIdentity)),
-                    cancellationToken);
+                // Step 5. Build install list using NuGet Resolver and select available resources.
+                List<SourcePackageDependencyInfo> availablePackagesToInstall;
+                if (allowMultipleVersions)
+                {
+                    availablePackagesToInstall = resolverContext.AvailablePackages.ToList();
+                }
+                else
+                {
+                    // Track packages which already installed and make sure only one version of package exists
+                    var resolver = new Resolver.PackageResolver();
+                    availablePackagesToInstall = await resolver.ResolveWithVersionOverrideAsync(resolverContext, project, DependencyBehavior.Ignore,
```

Comment:
Dependency behavior is changed here, which is not good

Reply:
oops, sorry
forgot to change it back